### ISSUE TITLE
Pressing escape doesn't close dialogs that use autofocus

### ISSUE THREAD

**Issue by robintown** [vector-im/element-web#20291]
### Steps to reproduce

1. Use Firefox (?)
2. Open the spotlight searcher, a feedback dialog, or any other dialog that autofocuses an input
3. Press escape

#### What did you expect?

The dialog should close.

#### What happened instead?

The dialog did not close.

### Operating system

NixOS unstable

### Browser information

Firefox 96.0

### URL for webapp

develop.element.io

### Application version

Element version: 840867bc0212-react-ec6bb8806822-js-a50a627300df
Olm version: 3.2.8

### Homeserver

townsendandsmith.ml

### Will you send logs?

No

**Comment by t3chguy** [comment]
Cannot reproduce this.

**Comment by t3chguy** [comment]
https://user-images.githubusercontent.com/2403652/147072202-8b1cd410-ecbc-4e63-9b7a-900520660706.mov

**Comment by robintown** [comment]
Weird, because I can still definitely reproduce this in Firefox 95.0.2

**Comment by robintown** [comment]
@t3chguy I determined that autofocus in focus locks is the root of the issue, hopefully the updated details help.

**Comment by robintown** [comment]
This seems to be fixed now, no idea why
